controllers/configurationpolicy_controller.go

```diff
@@ -1383,6 +1391,7 @@ func enforceByCreatingOrDeleting(obj singleObject, dclient dynamic.Interface) (r
 		} else {
 			reason = "K8s deletion success"
 			msg = fmt.Sprintf("%v %v existed, and was deleted successfully", obj.gvr.Resource, idStr)
+			obj.object = nil
 		}
```

Comment:
Am I looking at this right? Are these deletion `if` blocks backwards? It looks like if an object is deleted, `completed` is true and it has a "K8s deletion error"? But I thought things had been working, so I must be mistaken about something...